src/routes/(pages)/app/(app)/drip-lists/components/load-drip-lists.ts

```diff
@@ -0,0 +1,89 @@
+import { DRIP_LIST_CARD_FRAGMENT } from '$lib/components/drip-list-card/drip-list-card.svelte';
+import query from '$lib/graphql/dripsQL';
+import network from '$lib/stores/wallet/network';
+import { gql } from 'graphql-request';
+import {
+  DripListSortField,
+  SortDirection,
+  type QueryDripListsArgs,
+} from '$lib/graphql/__generated__/base-types';
+import type { SUPPORTED_CHAIN_IDS } from '$lib/stores/wallet/network';
+import FEATURED_DRIP_LISTS_CONFIG from './featured-drip-lists-config';
+import type { AllDripListsQuery, AllDripListsQueryVariables } from './__generated__/gql.generated';
+
+export const DRIP_LISTS_FRAGMENT = gql`
+  ${DRIP_LIST_CARD_FRAGMENT}
+  fragment DripLists on DripList {
+    ...DripListCard
+  }
+`;
+
+export const dripListsQuery = gql`
+  ${DRIP_LISTS_FRAGMENT}
+  query AllDripLists(
+    $where: DripListWhereInput
+    $sort: DripListSortInput
+    $chains: [SupportedChain!]
+    $limit: Int
+  ) {
+    dripLists(where: $where, sort: $sort, chains: $chains, limit: $limit) {
+      ...DripLists
+    }
+  }
+`;
+
+export function createDefaultFetchDripListsParameters(): QueryDripListsArgs {
+  return {
+    where: undefined,
+    sort: { direction: SortDirection.Asc, field: DripListSortField.MintedAt },
+    chains: [network.gqlName],
+    limit: undefined,
+  };
+}
+
+export async function fetchDripLists(f: typeof fetch, dripListsParameters?: QueryDripListsArgs) {
```

Comment:
I wanted to still figure out a way to cache this since it's called now in multiple places. I thought about perhaps creating a separate API endpoint that could take advantage of the redis cache. I tried to make something work that would make sense in and out of SSR, but no way josé.